docs/end2end-TrecQA-eval.md

```diff
@@ -54,22 +59,51 @@ Model to re-rank the passages (default: idf)
 -output (optional: file path)
 ```
 
-Path of the run file to be created
+Path of the run file, qrel file to be created (default:.)
 
 ```
 -qrel (optional: file path)
 ```
-Path of the qrel file to be created
+Path of an existing qrel file. A new qrel file will be created if this path is not passed.
+
+```
+-pattern (optional: pattern file path)
+```
+Path of the TREC13 QA pattern file `trec13factpats.txt`, which can be found 
+[here](http://trec.nist.gov/data/qa/2004_qadata/04.patterns.zip). The file path is a required argument if the type of 
+evaluation is pattern based.
 
 ```
-  --qa-model-file QA_MODEL_FILE
-                        the path to the model file
+-qa-model-file (Optional: SM model file path)
 ```
-Optional argument to choose a specific version of QA deep learning model to run ``run_trecqa.py`` with.
+This path is a required argument if the SM model is used as an answer extractor.
 
+```
+-w2v-cache (Optional: word2vec cache file path)
+```
+The path of the cache file is a required argument if the SM model is used as an answer extractor.
 
-The above command will create a run file in the `trec_eval` format and a qrel file
-called qrels.txt
+### Sample commands:
```

Comment:
Is there any reason that the example commands background the task? It might be surprising to the user if they just blindly copy-paste without examining what the command actually is. (More common behaviour than it should be)